agent/cvms/cvms.proto

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Ultraviolet
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+package cvms;
+
+option go_package = "./cvms";
+
+service CVMsService {
```

Comment:
Keep all caps: `CVMSService`, plural attaches

Reply:
We usually use plural for service naming, but since this is defined by package, we can simply call this `Service` so we have `github.com/absmach/cvms/Service` mnemonic.

Reply:
I am not sure... How did we for other services (Events for example)?

Reply:
It's called `events` - https://github.com/ultravioletrs/cocos/blob/main/agent/events - so we should either keep cvms here, or change in both.

Reply:
for protobuf we usually have service name followed by `Service` https://github.com/absmach/supermq/blob/d02d63d1eb8115f2da1ad67c68e389b4f9956b7e/internal/proto/auth/v1/auth.proto#L11